lib/mix/torch.ex

```diff
@@ -55,11 +55,11 @@ defmodule Mix.Torch do
     copy_from("priv/templates/#{format}/phx.gen.html", [
       {"controller_test.exs", "priv/templates/phx.gen.html/controller_test.exs"},
       {"controller.ex", "priv/templates/phx.gen.html/controller.ex"},
-      {"edit.html.#{format}", "priv/templates/phx.gen.html/edit.html.#{format}"},
```

Comment:
I don't follow these changes here?  Why does this need to be hard-code to remove the dynamic `format` extension?  They way I read the slime does, template files should be `*.html.slime` just like EEX templates are `*.html.eex`?